package-dry.json

```diff
@@ -1,8 +1,9 @@
 {
+  "name" : "backpack",
```

Comment:
For some reason npm was screaming at me when didn't have a name after I updated bindle.